src/api/emit.js

```diff
@@ -10,11 +10,16 @@ const CustomEvent = ((Event) => {
 })(window.CustomEvent);
 
 function createCustomEvent(name, opts = {}) {
-  if (CustomEvent) {
-    return new CustomEvent(name, opts);
+  let e;
+  if (Event) {
+    e = new Event(name, opts);
```

Comment:
The spread operator makes this nice, but needs a babel transform to support it.

I'll follow Bradley's approach.